vulnerable_people_form/integrations/data/local-area-shielding-advice-staging.csv

```diff
@@ -0,0 +1,2 @@
+ladcode,ladname,shielding_advice,start_datetime_utc
```

Comment:
it might be worth keeping staging the same as prod as it can help with testing different areas etc